starter-code/src/components/button/Button.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import contacts from '../../../src/contacts.json';
+
+const Button = ({actors, actorName, name, isAdd, isName, isPop, isDel, updateState}) => {
+  const action = () => {
+    if (isAdd) {
+      const randomActor = contacts.splice(Math.floor(Math.random() * contacts.length), 1);
+      const newActors = actors.push(randomActor);
+      updateState(newActors);
+    } else if (isName) {
+      const sortedActors = actors.sort((a, b) => {
+        if (a.name[0] > b.name[0]) {
```

Comment:
Usar o `localeCompare()` pois quando a primeira letra do nome for igual, haverá problema.